ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -55,6 +55,11 @@ tests:
   openshift_installer:
     cluster_profile: gcp
     upgrade: true
+- as: e2e-sdn-step-registry
+  commands: ""
+  steps:
+    cluster_profile: gcp
```

Comment:
@pecameron gcp -> aws (as you pointed out, no gcp support in step registry yet)

Reply:
fixed

Reply:
fixed